boards/nucleo-f303/include/periph_conf.h

```diff
@@ -0,0 +1,259 @@
+/*
+ * Copyright (C) 2015  Freie Universität Berlin
+ * Copyright (C) 2015 Hamburg University of Applied Sciences
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_nucleo-f303 Nucleo-F303
+ * @{
+ *
+ * @file
+ * @brief       Peripheral MCU configuration for the nucleo-f303 board
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author      Katja Kirstein <katja.kirstein@haw-hamburg.de>
+ */
+
+#ifndef __PERIPH_CONF_H
+#define __PERIPH_CONF_H
```

Comment:
Change this include guard to `PERIPH_CONF_H_` instead. Double underscores are reserved.